### PR TITLE
Create new rit delete env command

### PR DESCRIPTION
**Pull request by kaduartur**
### Description
The `rit delete context`  command has been deprecated and we will now use the new `rit delete env` command which has the same functionality as the old command.

### How to verify it
It's a small part of the #554 issue, a separate pull request was created to facilitate code review

### CODE REVIEW

File: pkg/cmd/delete_env_test.go [hunk 1]
```diff
@@ -18,13 +18,19 @@ package cmd
 
 import (
 	"testing"
+
+	"github.com/ZupIT/ritchie-cli/pkg/env"
 )
 
 func TestNewDeleteContextCmd(t *testing.T) {
```
Comment (brunasilvazup) [inline pkg/cmd/delete_env_test.go:25]: Typo here?

File: pkg/cmd/mocks_test.go [hunk 2]
```diff
@@ -217,14 +217,16 @@ func (envFinderMock) Find() (env.Holder, error) {
...
 
-func (ctxFindRemoverMock) Find() (env.Holder, error) {
-	f := envFinderMock{}
-	return f.Find()
+func (e envFindRemoverMock) Find() (env.Holder, error) {
+	return e.holder, e.err
 }
 
-func (ctxFindRemoverMock) Remove(ctx string) (env.Holder, error) {
+func (envFindRemoverMock) Remove(ctx string) (env.Holder, error) {
```
Comment (brunasilvazup) [inline pkg/cmd/mocks_test.go:229]: `ctx` to `env`?

**Review by brunasilvazup — APPROVED** [review]

File: pkg/cmd/delete_env_test.go (hunk 1 above)
Comment (kaduartur) [reply pkg/cmd/delete_env_test.go:25]: I will rename all variables and functions in another PR

File: pkg/cmd/mocks_test.go (hunk 2 above)
Comment (kaduartur) [reply pkg/cmd/mocks_test.go:229]: same case here

**Pull request merged**